src/App.js

```diff
@@ -0,0 +1,131 @@
+import './App.css';
+import React, {useState} from 'react';
+import Counter from "./assets/Components/fruitCounter";
+
+
+function App() {
+    const [bananaCounter, setBananaCounter] = useState(0)
+    const [aambeienCounter, setAambeienCounter] = useState(0)
+    const [appelCounter, setAppelCounter] = useState(0)
+    const [kiwiCounter, setKiwiCounter] = useState(0)
```

Comment:
De set functies worden niet in het Counter component aangeroepen, omdat die daar zelf de state bijhouden. Daarom zie je ook bij de submit dat alle counters nog op 0 staan in de console. Je zou de counter en de set-functie mee kunnen geven aan het component om het werkend te krijgen